test/lib/commands/access.js

```diff
@@ -13,20 +14,21 @@ t.test('completion', async t => {
     t.resolves(res, expect, argv.join(' '))
   }
 
-  testComp(['npm', 'access'], [
-    'public', 'restricted', 'grant', 'revoke', 'ls-packages',
-    'ls-collaborators', 'edit', '2fa-required', '2fa-not-required',
+  testComp(['npm', 'access'], ['list', 'get', 'set', 'grant', 'revoke'])
+  testComp(['npm', 'access', 'list'], ['packages', 'collaborators'])
+  testComp(['npm', 'access', 'ls'], ['packages', 'collaborators'])
```

Comment:
`npm owner add` suggests `s/collaborators/owners`?

Reply:
They are interacting with totally different api endpoints. One adds/removes folks from `maintainers` packument, the other interacts with a `/-/package/[name]/collaborators` endpoint.  There's obviously overlap there but we are providing access to the two api methods we have.

Reply:
I’m just speaking to the terminology - user-facing terms should be consistent even if api endpoints weren’t built to be.